e2e/testcases/legacy_test.go

```diff
@@ -174,9 +174,6 @@ func TestBats(t *testing.T) {
 		t.Skip("Skip running bats test on non-local git providers")
```

Comment:
Since this PR converts all the remaining bats tests to golang tests, we may remove this file and all the bats test files.

Reply:
Sounds good. Thasnks!